lib/openssl/signature_algorithm/eddsa.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "ed25519"
+require "openssl/signature_algorithm/base"
+
+module OpenSSL
+  module SignatureAlgorithm
+    class EdDSA < Base
+      class Ed25519
```

Comment:
In the same vein as the curve argument comment, we can remove this class from the namespace for now.